### PR TITLE
Konstantinos / Responsive issue in tradertools

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by mitra-deriv** [comment]
Hi @konstantinosG-deriv seems the issue is not fixed, I checked your testlink on Iphone 12 it looks cropped 😞 

IMG_7049

**Review by habib-deriv — APPROVED** [review]

File: src/pages/trader-tools/common/_style.ts [hunk 1]
```diff
@@ -120,6 +119,8 @@ export const FormWrapper = styled(Flex)`
 
     @media ${device.laptopM} {
         margin: 0 0 40px;
+        overflow: scroll;
```
[the remark this answers is not in the document]
Comment (konstantinosG-deriv) [reply src/pages/trader-tools/common/_style.ts:122]: Ok 👍

**Review by habib-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

File: src/pages/trader-tools/common/_style.ts [hunk 2]
```diff
@@ -40,43 +40,38 @@ export const StyledHeader = styled(Header)`
...
-    display: flex;
-    justify-content: center;
-    align-items: center;
-
-    @media (min-width: 1400px) {
-        padding: 0 20%;
+    width: 79.2rem;
+    margin: 0 auto;
+    line-height: 1.5;
+    @media ${device.tablet} {
```
Comment (prince-deriv) [inline src/pages/trader-tools/common/_style.ts:72]: Can we add a space before `@media` here.  Also why do we need the `79.2rem` width?

Comment (konstantinosG-deriv) [reply src/pages/trader-tools/common/_style.ts:72]: Yes will add a space, idk about the 79.2rem it was already like this in codebase. I had to start from the beginning so I changed back to the original settings most of the lines

**Pull request closed without merging**